ads/google/adsense.js

```diff
@@ -29,6 +31,15 @@ export function adsense(global, data) {
       ['adClient', 'adSlot', 'adHost', 'adtest', 'tagOrigin', 'experimentId',
         'ampSlotIndex', 'adChannel', 'autoFormat', 'fullWidth']);
 
+  if (data['autoFormat'] === 'rspv') {
```

Comment:
You can remove the if statement entirely by combining rspv check in assert:

user().assert(data['autoFormat'] != 'rspv' || data['height] == ADSENSE_RSPV_WHITELISTED_HEIGHT, ...)

Also do you want == or === when doing height comparison?  Do you know it will always be a number as opposed to a string?